test/__snapshots__/Base64.spec.ts.snap

```diff
@@ -1,32 +1,32 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Base64 #encode gas cost of encode(<some html>) 1`] = `1497`;
+exports[`Base64 #encode gas cost of encode(<some html>) 1`] = `1656`;
 
-exports[`Base64 #encode gas cost of encode(aLpHaBeT) 1`] = `1256`;
+exports[`Base64 #encode gas cost of encode(aLpHaBeT) 1`] = `1435`;
 
-exports[`Base64 #encode gas cost of encode(alphabet soup) 1`] = `1727`;
+exports[`Base64 #encode gas cost of encode(alphabet soup) 1`] = `1866`;
 
-exports[`Base64 #encode gas cost of encode(f) 1`] = `763`;
+exports[`Base64 #encode gas cost of encode(f) 1`] = `982`;
 
-exports[`Base64 #encode gas cost of encode(fo) 1`] = `774`;
+exports[`Base64 #encode gas cost of encode(fo) 1`] = `993`;
 
-exports[`Base64 #encode gas cost of encode(foo) 1`] = `769`;
+exports[`Base64 #encode gas cost of encode(foo) 1`] = `988`;
 
-exports[`Base64 #encode gas cost of encode(foob) 1`] = `1004`;
+exports[`Base64 #encode gas cost of encode(foob) 1`] = `1203`;
 
-exports[`Base64 #encode gas cost of encode(fooba) 1`] = `1015`;
+exports[`Base64 #encode gas cost of encode(fooba) 1`] = `1214`;
 
-exports[`Base64 #encode gas cost of encode(foobar) 1`] = `1010`;
+exports[`Base64 #encode gas cost of encode(foobar) 1`] = `1209`;
 
 exports[`Base64 #encode gas cost of encode(includes
-newlines) 1`] = `1979`;
+newlines) 1`] = `2098`;
 
-exports[`Base64 #encode gas cost of encode(test string) 1`] = `1497`;
+exports[`Base64 #encode gas cost of encode(test string) 1`] = `1656`;
 
-exports[`Base64 #encode gas cost of encode(this is a test) 1`] = `1738`;
+exports[`Base64 #encode gas cost of encode(this is a test) 1`] = `1877`;
 
-exports[`Base64 #encode gas cost of encode(this is a very long string that should cost a lot of gas to encode :)) 1`] = `6083`;
+exports[`Base64 #encode gas cost of encode(this is a very long string that should cost a lot of gas to encode :)) 1`] = `5862`;
 
-exports[`Base64 #encode gas cost of encode(😀) 1`] = `1004`;
+exports[`Base64 #encode gas cost of encode(😀) 1`] = `1203`;
```

Comment:
Guessing this is all due to external library?

Reply:
yeah